core/models/TransactionModels.py

```diff
@@ -12,15 +12,17 @@ def validate_auth(authorizer):
     # TODO: should this be given a second argument of min status level?
 
     # If the member is not a staffer, then they are not allowed to authorize a transaction like this
-    if not authorizer.is_staff:
-        raise ValidationError("{} is not allowed to authorize a transaction".format(authorizer.name))
+    required_perm = 'authorize_transactions'
+    if not authorizer.has_permission(required_perm):
+        raise ValidationError("{} is not allowed to authorize a transaction".format(authorizer.get_short_name))
 
 
-def validate_can_checkout(member):
+def validate_can_rent(member):
```

Comment:
I think the name of this method implies that it returns True/False

Reply:
arguable, but I don't really have a better idea for a name